docs/iota.go/reference/api_attach_to_tangle.md

```diff
@@ -0,0 +1,53 @@
+# API -> AttachToTangle()
+AttachToTangle performs the Proof-of-Work required to attach a transaction to the Tangle by calling the attachToTangle IRI API command. Returns a list of transaction trytes and overwrites the following fields:  Hash, Nonce, AttachmentTimestamp, AttachmentTimestampLowerBound, AttachmentTimestampUpperBound  If a Proof-of-Work function is supplied when composing the API, then that function is used instead of using the connected node.
```

Comment:
Reckon some of these formatting things are caused by the way things are put together after parsing?:
"fields:  Hash,"
extra space
"AttachmentTimestampUpperBound  If a Proof-of-Work"
missing period after AttachmentTimestampUpperBound

Not a big deal, so if it's a hassle to fix then not to worry.

Reply:
Fixed. The comments on the function were wrong.